api/app/applicationModel/api.py

```diff
@@ -171,7 +171,7 @@ def put(self):
                 section.update(new_s)
                 db.session.commit()
 
-        question = app_repository.get_questions_by_id(question_id) # Confirm correct ID
+        question = app_repository.get_questions_by_id(section.id, app_form.id) # Confirm correct ID
```

Comment:
I'd just like to confirm that I understand - is the question is identified by the corresponding section ID and app form ID?
I'm wondering if it's necessary to add a method to "get" the question based on these factors, or if there's a way to get the existing question ID I am missing.